doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
-   * any RA couples to RM
-   * like RI couples to RA
-   * like VIs (Vendor's Implementations) couple to RA
-   * Operators can design or procure a compatible fabric
-1. Concurrently supports containerized and virtualized coexistence for VNF->CNF cutovers, as well as protracted parallel operations
-1. Provide a version controlled catalog of APIs, and their respective spans of control, capabilities and purpose, to facilitate predictable integration w/ a wide selection of fabric implementations
-1. Provide ability for any number of Operator-specific fabrics to power CNTT cloud infrastructure
-1. Enable RC's ability to realize mandated OVP qualification deliverables
-1. Unambiguously document the responsibilities of each CNTT constituency
-1. In cases where a VNF/CNF require HW layer resources it should be under the control of the Virtualization Layer
-1. It is important that the HW Infrastructure Manager, each VIM and each VNF/CNF could be managed by separate organizations
-1. Drive the industry towards convergence on ABIs supporting Cloud-Native implementations for SmartNICs
-
-> _**Editor's Note:** Consider moving objectives to a table_
+### 5.1.2 Networking Principles & Strategy
+
+This section introduces some networking related principles that should be followed during development and definition of Networking scopes for Reference Model, Reference Architecture 01 (OpenStack), Reference Architecture 02 (K8s) and down to Reference Implementations and Reference Conformance test suites under CNTT.
+
+•	**Abstraction**: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
+> **Note:** Abstraction may be deployed in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms. This includes Network Functions running on server nodes with or without programmable hardware acceleration, or networking support functions running on programmable network switches in the network.
+
+•	**Agnosticity** : Define Network Fabric concepts and models that can carry any type of traffic in terms of:
+-	Control, User and Management traffic types
+-	Acceleration technologies supporting multiple types of Infrastructure deployments and Network Function workloads 
+
+•	**Automation**: Enable full automation, from Physical Fabric installation and provisioning to automation of VNF/CNF onboarding.
+
+•	**Openness**: All Networking is based on open source or standardized APIs (NBI and SBI) and should enable integration of Open Source networking components (e.g. SDN controllers).
+
+•	**Programmability**: Should enable a programmable forwarding plane controlled from a separately deployed control plane that abstracts into standardized functions and interfaces.
```

Comment:
Centralized vs. Distributed fabric is an implementation detail. Propose we change "Should" -> "May", to provide latitude for Operators who decide to go Distributed.